docs/javascript/v2/guides/javascript-quickstart.mdx

```diff
@@ -3,154 +3,306 @@ title: JavaScript Quickstart Guide
 nav: 2.1
 ---
 
-## Getting started
+## Overview
 
-Hello there! In this guide, we'll build a video conferencing application using the 100ms JavaScript SDK. This guide doesn't make any assumptions related to
-the framework or library choices. In case you plan to use React, do check our React quickstart as well.
+Hello there, In this guide, we'll walk you through step-by-step instructions to create a video conferencing application using the 100ms JavaScript SDK and test it locally on your device. In case you plan to use React, do check our [React quickstart](https://www.100ms.live/docs/javascript/v2/guides/react-quickstart) as well.
 
-TL;DR - You can find the link to the complete example over [here](https://codesandbox.io/s/hms-js-quickstart-5rmes). There is also a version of the quickstart using CDN [here](https://codepen.io/triptu/pen/dymxmoR?editors=0010).
+TL;DR - Try out this complete example in [CodeSandbox](https://codesandbox.io/s/hms-js-quickstart-5rmes).
 
-### Prerequisites
+## Setup a local Javascript app
 
-To get started you must be familiar with rudimentary JavaScript and have [nodejs](https://nodejs.org/) installed on your machine.
+In this walk-through, we will use [ParcelJS](https://parceljs.org/getting-started/webapp/) a web application bundler that requires zero configuration. If you wish to use any other library or framework, you can set up your app using that.
 
-### Installing the dependencies
+### Initialize a project
+
+You can initialize a project using `npm init` or `yarn init`. While initializing the project you can enter the details for your project as you wish.
+
+### Install ParcelJS & 100ms SDK
+
+Then, install ParcelJS and 100ms SDK into your app using yarn or npm like below:
 
 ```bash section=InstallingTheDependencies sectionIndex=1
-npm install --save @100mslive/hms-video-store@latest
+yarn add --dev parcel @100mslive/hms-video-store
 ```
 
-### Initializing the SDK
+or
 
-Let us start with initializing the libraries. We need an instance of `HMSStore` and `HMSActions` to get started. Copy the code snippet below and paste it into your project.
+```bash section=InstallingTheDependencies sectionIndex=2
+npm install --save-dev parcel @100mslive/hms-video-store
+```
+
+### Project setup
 
-```jsx:script.js section=InitializingTheSDK sectionIndex=1
-import { HMSReactiveStore } from '@100mslive/hms-video-store';
+Now that ParcelJS is installed, let's create basic source files for our video-conferencing application. Let’s create the files and folders in the below manner:
```

Comment:
should we mention the parceljs here? Now that the depedencies are installed should be fine I think

Reply:
updated it.